### ISSUE TITLE
How to disable common info logs?

### ISSUE THREAD

**Issue by william-keller** [ikyriak/IdempotentAPI#25]
How to disable common info logs?

My logs are being long and difficult to reach relevant messages.

Is there a way to disable just those info logs? keeping warning and error logs just when they happen

**Comment by ikyriak** [comment]
Hello @william-keller,

I have made some changes to check the logging level and improve logging performance.
You can test the implementation by using the [0.3.0-prerelease-01](https://www.nuget.org/packages/IdempotentAPI/0.3.0-prerelease-01) version.

Now we can set the logging level in our `appsettings.json` file, such as:
```json
{
  "Logging": {
    "LogLevel": {
      "Default": "Information",
      "IdempotentAPI.Core.Idempotency": "Warning"
    }
  }
}
```

**Comment by ikyriak** [comment]
Hello @william-keller,

The related improvement has been included in the [1.0.0 RC.01](https://github.com/ikyriak/IdempotentAPI/pull/30) release.

Please read the [README.md](https://github.com/ikyriak/IdempotentAPI#-quick-start) and [CHANGELOG.md](https://github.com/ikyriak/IdempotentAPI/blob/master/CHANGELOG.md) files because this release has breaking changes.

You can find the 1.0.0 RC.01 NuGet packages [here](https://www.nuget.org/packages?q=IdempotentAPI).

If you have any suggestions or comments please, do not hesitate to send them 😉
